SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -167,17 +176,21 @@ bool FSpatialGDKEditor::GenerateSchema(ESchemaGenerationMethod Method)
 												 *ProjectPath, FApp::IsEngineInstalled() ? TEXT(" -installed") : TEXT(""), *ProjectPath,
 												 *FUnrealEdMisc::Get().GetExecutableForCommandlets(), *OptionalParams);
 
-		IUATHelperModule::Get().CreateUatTask(
-			UATCommandLine, FText::FromString(PlatformName), LOCTEXT("CookAndGenerateSchemaTaskName", "Cook and generate project schema"),
-			LOCTEXT("CookAndGenerateSchemaTaskShortName", "Generating Schema"), FEditorStyle::GetBrush(TEXT("MainFrame.PackageProject")));
+		TFunction<void(FString, double)> Callback = [Result](FString UATResult, double) {
```

Comment:
Can UATResult be a const FString&?

Reply:
I think so, lambda bindings in Unreal is a mystery to me but I think it can be both. Will change